src/views/Pools/components/CardFooter.tsx

```diff
@@ -94,6 +109,10 @@ const CardFooter: React.FC<Props> = ({
   const blocksUntilStart = Math.max(startBlock - block, 0)
   const blocksRemaining = Math.max(endBlock - block, 0)
 
+  const isMetamask = (window as WindowChain).ethereum && (window as WindowChain).ethereum.isMetaMask && tokenAddress
```

Comment:
there is a miss match between your variable name, and what it really is: `isMetamask` and `tokenAddress`

Reply:
Another reason to add the types...

Reply:
Can you remove the `isMetaMask` check so we can test if this api works on other wallets ?

Reply:
Sure